dogm/demo/simulator/test/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+SET(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+
+ADD_EXECUTABLE (simulator_spec simulator_spec.cpp)
+TARGET_LINK_LIBRARIES (
+    simulator_spec
```

Comment:
Consider adding this spec to CI. I'm actually thinking about using ctest (see [local_pipeline](https://github.com/TheCodez/dynamic-occupancy-grid-map/blob/master/local_pipeline_ubuntu.sh#L40)) for this. I would be happy to refactor the ubuntu CI workflow to include this test and use ctest in a follow-up PR.

Reply:
Yes sure go ahead :)